astroquery/alma/tests/test_alma_remote.py

```diff
@@ -59,8 +54,8 @@ def fin():
         request.addfinalizer(fin)
         return my_temp_dir
 
-    def test_public(self):
-        alma = get_client()
+    def test_public(self, alma):
+        assert alma.archive_url == 'https://almascience.eso.org'
```

Comment:
isn't this going to depend on where you run the tests?  Or am I missing the part where eso is the default?

Reply:
Doh! That has no place there - just a leftover from my debugging. Thanks for spotting it.